.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,15 @@
+name: WCAG tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  call_wcag_test:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/wcag_test.yml@v1
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    with:
+      build_script:
+        'npm run storybook & while ! curl http://localhost:6006 -I; do sleep 1; done'
```

Comment:
instead of downgrading jest for the whole repo would it be possible to manually install jest@27 just for the wcag tests as part of the workflow?

Reply:
I imagined it would also be nice to be able to run the wcag command locally to test, which would mean we would need to install v27 locally for that too. and I think we'd also need to do the same for the visual coverage stuff. so it seemed easier to downgrade the whole repo for now

Reply:
that's fair!